### ISSUE TITLE
Finish up the features/budget branch and merge it back into development

### ISSUE THREAD

**Issue by jmcameron** [IMA-WorldHealth/bhima#358]
The features/budget branch provides (1) a prototype implementation of the basic budget/balance page for all income/expense account and (2) a new page for editing or creating budget data for specific accounts.  

The budget/balance page includes a function to export a CSV file of the raw data but it is not quite working yet.  See Issue https://github.com/IMA-WorldHealth/bhima/issues/356.

These budget-related pages need to be tested and updated as necessary, and the CSV export function needs completing.  Once that is done, the features/budget branch should be merged into development.

**Comment by jmcameron** [comment]
Completed in PR https://github.com/IMA-WorldHealth/bhima/pull/367
